app/src/main/java/jp/developer/retia/frozenword/ui/activity/AddHabbitActivity.kt

```diff
@@ -72,29 +93,70 @@ class AddHabbitActivity : ComponentActivity() {
     }
 }
 
+@Composable
+fun AddHabbitScreen(addHabbitViewModel: AddHabbitViewModel = viewModel()){
```

Comment:
```suggestion
fun AddHabbitScreen(addHabbitViewModel: AddHabbitViewModel = viewModel()) {
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>